econsa/tests/test_correlation.py

```diff
@@ -0,0 +1,136 @@
+"""Test for correlation transformation."""
+import chaospy as cp
+import numpy as np
+import pytest
+from scipy.stats import norm
+from statsmodels.stats.correlation_tools import corr_nearest
+
+from econsa.copula import _cov2corr
+from econsa.correlation import gc_correlation
+
+
+def get_strategies(name):
+    dim = np.random.randint(2, 10)
+    means = np.random.uniform(-100, 100, dim)
+
+    if name == "test_gc_correlation_functioning":
+        # List of distributions to draw from.
+        # Repeated distributions are for higher drawn frequency, not typo
```

Comment:
We agreed to not use the duplications here, why is that still the case?

Reply:
Sorry I missed that, correcting now…